.gitignore

```diff
@@ -13,3 +13,5 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+**/bin/
+.vscode/*
```

Comment:
Could you please add below too.

# Trash files
*\.DS_Store

Sorry MAC files hehe